src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -3731,8 +3731,8 @@ You should consider suppressing the warning only if you're sure that you don't w
   <data name="WRN_IsDynamicIsConfusing_Title" xml:space="preserve">
     <value>Using 'is' to test compatibility with 'dynamic' is essentially identical to testing compatibility with 'Object'</value>
   </data>
-  <data name="ERR_OverloadRefOutCtor" xml:space="preserve">
-    <value>Cannot define overloaded constructor '{0}' because it differs from another constructor only on ref and out</value>
+  <data name="ERR_OverloadRefKindCtor" xml:space="preserve">
+    <value>Cannot define overloaded constructor '{0}' because it differs from another constructor only on parameter reference kind</value>
```

Comment:
Still trying to decide whether this is the best wording of the error message, or should be detailed to:
`Cannot define overloaded constructor '{0}' because it differs from another constructor only on modifiers {1} and {2}`